widget/view.controller.js

```diff
@@ -0,0 +1,89 @@
+/* Copyright start
+  Copyright (C) 2008 - 2023 Fortinet Inc.
+  All rights reserved.
+  FORTINET CONFIDENTIAL & FORTINET PROPRIETARY SOURCE CODE
+  Copyright end */
+  'use strict';
+  (function () {
+    angular
+      .module('cybersponse')
+      .controller('vtAugment100Ctrl', vtAugment100Ctrl);
+    vtAugment100Ctrl.$inject = ['$scope', '$sce', '$resource', 'API', '_', 'toaster', '$rootScope', '$state', 'Entity'];
+  
+    function vtAugment100Ctrl($scope, $sce, $resource, API, _, toaster, $rootScope, $state, Entity) {
+      $scope.renderingURL = { url: '' }
```

Comment:
url can be directly assigned to the variable renderingURL instead of creating object with single key-value

Reply:
done